requirements.txt

```
@@ -1,6 +1,6 @@
 mkdocs==1.5.3
 mike==1.1.2
-markdown==3.3.7 # Temporarily pin version <3.4, see https://github.com/mkdocs/mkdocs/pull/2893
+markdown==3.5 # Temporarily pin version <3.4, see https://github.com/mkdocs/mkdocs/pull/2893
```

Removing the old comment.

```suggestion
markdown==3.5
```